### PR TITLE
feat(swift): add snippets

### PR DESCRIPTION
**Pull request by Fluf22**
## 🧭 What and Why

🎟 JIRA Ticket: [DI-1793]

### Changes included:

- add snippets gen

[DI-1793]: https://algolia.atlassian.net/browse/DI-1793?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: snippets/swift/Package.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        .package(path: "../../clients/algoliasearch-client-swift"),
+        .package(url: "https://github.com/Flight-School/AnyCodable", .upToNextMajor(from: "0.6.1"))
+    ],
+    targets: [
+        .target(
+            name: "AlgoliaSearchClientSnippets",
+            dependencies: [
+                .product(name: "AnyCodable", package: "AnyCodable"),
+                .product(
+                  name: "Abtesting",
```
Comment (millotp) [inline snippets/swift/Package.swift:28]: you can generate this file right ?

**Review by millotp — COMMENTED** [review]
I don't think the formatter is running on the generated snippet ahah

File: snippets/swift/Package.swift (hunk 1 above)
Comment (Fluf22) [reply snippets/swift/Package.swift:28]: Yes

**Review by millotp — APPROVED** [review]
gg !

**Pull request merged**